src/make.ts

```diff
@@ -244,11 +240,17 @@ export function makeBoolean() {
   return checkBoolean;
 }
 
-export function makeArray(maker: any) {
+export function makeArray(maker: any, minSize?: number, maxSize?: number) {
   return (value: any, opts?: MakeOptions) => {
     if (!Array.isArray(value)) {
       return getErrorWithValueMsg('expected an array', value);
     }
+    if (minSize != null && value.length < minSize) {
```

Comment:
openapi doesn't specify if this value is inclusive or exclusive 🤷‍♂️

Reply:
they say its same semantics as json-schema which seems to have it inclusive https://json-schema.org/understanding-json-schema/reference/array.html#length